Code/Framework/AzNetworking/AzNetworking/Utilities/Endian.h

```diff
@@ -14,14 +14,14 @@
 #include <AzNetworking/Utilities/Endian_Platform.h>
 
 #if AZ_TRAIT_NEEDS_HTONLL
-static const uint64_t htonll(uint64_t value)
+inline static const uint64_t htonll(uint64_t value)
```

Comment:
You should replace `static` with `inline` instead of appending it.
That way the function has external linkage and only one copy gets added to shared library